### ISSUE TITLE
bkp s -m 'a nice message'

### ISSUE THREAD

**Issue by deonthomasgy** [Elive/elive-tools#1]
i am requesting you to create a new feature for this tool, so i can save messages when i create a save using "bkp s" 
e.g
bkp s -m 'before merge'
bkp s -m 'after merge' 

-m [the message you want to save]

**Comment by TheTechRobo** [comment]
Seven years later, this feature would still be nice.

@Thantermesis , are you listening?

**Comment by Thanatermesis** [comment]
yeah sure, the only problem is that I have many work around and this is just one more thing in the list lol

**Comment by TheTechRobo** [comment]
Yeah that makes sense :)
I see you read my article! Thanks for that !


Sent from TtR's iPhone 4

Le 9 mars 2020 à 15:40, "Thanatermesis" <notifications@github.com<mailto:notifications@github.com>> a écrit :


yeah sure, the only problem is that I have many work around and this is just one more thing in the list lol
